test_room_class.py

```diff
@@ -1,87 +1,152 @@
 import unittest
-from room import Room
+from room import Room, Office
 from people import Person, Fellow, Staff
 from allocator import Dojo
 
-
 class TestRoomClass(unittest.TestCase):
     def setUp(self):
         self.test_dojo = Dojo()
-        self.fellow1 = Fellow()
+        self.fellow1 = Fellow("bryan")
         self.is_office = True
+        self.need_room = True
         self.is_fellow = True
 
+        self.room1 = Room()
+        self.office1 = Office("blue_office")
+        #
         """ creating multiple offices"""
-        self.three_rooms = self.test_dojo.create_room(self.is_office, ['Blue', 'Black', 'Yellow'])
-
-        """ creating an office"""
-        self.one_room = self.test_dojo.create_room(self.is_office, ['Spires'])
+        self.test_dojo.create_room(self.is_office, ['Spires', 'Black', 'Yellow'])
 
-        """ Add a person"""
-        one_person = self.test_dojo.create_person("bryant", "neymar", self.is_fellow)
+        """ creating multiple living spaces"""
```

Comment:
This should be a comment not a doc string. Doc strings are used to describe what a class/function does.